src/ChainSafe.Gaming/RPC/Events/WebSocketEventManager.cs

```diff
@@ -14,6 +14,10 @@
 
 namespace ChainSafe.Gaming.RPC.Events
 {
+    /// <summary>
+    /// <see cref="ILifecycleParticipant.WillStartAsync"/> will be executed later.
+    /// </summary>
+    [ExecutionOrder(10)]
```

Comment:
why is this an attribute and not a member that you then just use for sorting services?

Reply:
it's essentially the same thing but this is less invasive, we just have to put this attribute and if it's not present it'll default the execution order to 0.

Reply:
redone